src/rngs/std.rs

```diff
@@ -66,8 +61,8 @@ impl SeedableRng for StdRng {
     }
 
     #[inline(always)]
-    fn from_rng<R: RngCore>(rng: R) -> Result<Self, Error> {
-        Rng::from_rng(rng).map(StdRng)
+    fn from_rng<R: RngCore>(rng: R) -> Self {
+        Self(Rng::from_rng(rng))
     }
```

Comment:
The default impl is enough? IIRC this was impl'd explicitly since the old previous impl had an `unwrap` that we didn't need here.